source/physics/elasticity/standard_tensors.inst.in

```diff
@@ -0,0 +1,41 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+
+for (deal_II_dimension : DIMENSIONS)
+{
+    template
+    class StandardTensors<deal_II_dimension>;
```

Comment:
I believe that when I instantiate a class instance, all of the static members will be instantiated as well (in this instance, `StandardTensors<dim>::I` etc. It certainly seems to work this way on my machine (clang 3.7). Is this correct in general, or should I rather instantiate each static member individually.